config/core/deployments/activator-hpa.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```

Comment:
We need to bump manifests for CI. Could you please run:

```
 ./openshift/release/generate-release.sh knative-v1.8.0
 ./openshift/release/generate-release.sh ci
```

? These will bump `openshift/release/knative-serving-*` files. 

Also, we need to update the manifests in `openshift/release/artifacts/*` for SO repo.